apps/platform/src/app/config/config.ts

```diff
@@ -1,3 +1,33 @@
 // link to external form for submitting projects
 export const CodaFormProjectLink =
 	'https://coda.io/form/Zazelenimo-Split-obrazac-za-prijedloge_dbtTs0gGIcq';
+
+// List of Split city neighborhoods
+export const neighborhoods = [
+	'Varoš',
+	'Bačvice',
+	'Meje',
+	'Pujanke',
+	'Žnjan',
+	'Sućidar',
+	'Kman',
+	'Spinut',
+	'Kocunar',
+	'Bol',
+	'Gripe',
+	'Lovret',
+	'Trstenik',
+	'Visoka',
+	'Mertojak',
+	'Brda',
+	'Plokite',
+	'Skalice',
+	'Manus',
+	'Firule',
+	'Šine',
+	'Smrdečac',
+	'Sirobuja',
+	'Stobreč',
+	'Žrnovnica',
+	'Vranjic',
```

Comment:
Order alphabetically, unless you need this order.
```suggestion
		'Bačvice',
		'Bol',
		'Brda',
		'Firule',
		'Gripe',
		'Kman',
		'Kocunar',
		'Lovret',
		'Manus',
		'Meje',
		'Mertojak',
		'Plokite',
		'Pujanke',
		'Šine',
		'Sirobuja',
		'Skalice',
		'Smrdečac',
		'Spinut',
		'Stobreč',
		'Sućidar',
		'Trstenik',
		'Varoš',
		'Visoka',
		'Vranjic',
		'Žnjan',
		'Žrnovnica',
```